python/perf-kernels/flash-attention.py

```diff
@@ -387,6 +420,19 @@ def attn_fwd(
         )
     else:
         bias_ptr = None
+
+    if USE_ALIBI != 0:
+        alibi_slopes_ptr = tl.make_block_ptr(
```

Comment:
I think we don't need a block pointer for this. IIUC, for a given head / batch, the entire QK result uses the same scalar alibi value which would mean each instance of this kernel is loading a scalar. We can use an old school `tl.load(offset)` for that.

Reply:
Done